stackstorm/Dockerfile

```diff
@@ -1,16 +1,16 @@
 FROM debian:wheezy
 
 ARG ST2_VERSION
-ARG ST2_REPO=stable
+ARG ST2_REPO=staging-stable
 ARG ST2_PACKAGE=https://packagecloud.io/StackStorm/${ST2_REPO}/packages/debian/wheezy/st2_${ST2_VERSION}_amd64.deb/download
 
 LABEL com.stackstorm.version="${ST2_VERSION}"
 
-ADD ./sources.list /etc/apt/ 
+ADD ./sources.list /etc/apt/
 
-RUN apt-get -y update && apt-get -y install adduser procps python2.7 libffi5 libyaml-0-2 git libpcre3 libpython2.7 libxml2 python-dev libssl-dev libffi-dev sudo
+RUN apt-get -y update && apt-get -y install wget adduser procps python2.7 libffi5 libyaml-0-2 git libpcre3 libpython2.7 libxml2 python-dev libssl-dev libffi-dev sudo
 
-ADD ${ST2_PACKAGE} /tmp/st2_${ST2_VERSION}_amd64.deb
+RUN wget ${ST2_PACKAGE} -O /tmp/st2_${ST2_VERSION}_amd64.deb
```

Comment:
`ADD` works fine too for remote URLs: https://docs.docker.com/engine/reference/builder/#add